hcn/hcnsupport_test.go

```diff
@@ -23,3 +23,10 @@ func TestV2ApiSupport(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func TestRemoteSubnetSupport(t *testing.T) {
```

Comment:
You need to base this on some host version. Older versions of Windows (RS5) will fail it right?

Reply:
The previous code would've returned a platformnotsupported error if the HNS version on the machine was too low. I was following the model of the V2ApiSupported test, but I understand why it's an issue. I've updated the test now

Reply:
Thanks